elm/Main.elm

```diff
@@ -115,27 +117,42 @@ connect url =
         }
 
 
-rootPane : MPane
-rootPane =
+fileRootPane : MPane
+fileRootPane =
     Pane.new "root" (Pane.loading "/") <| Mpd.encodeCmd <| CmdLoadDir "root" ""
 
 
-artistPane : MPane
-artistPane =
-    Pane.new "artists" (Pane.loading "Artist") <| Mpd.encodeCmd <| CmdList "artists" { what = "artists", artist = "", album = "" }
+artistRootPane : Bool -> MPane
+artistRootPane useAlbumartist =
```

Comment:
I think the elm part of this would be a bit more readable/robust with a dedicated type:

```
type ArtistChoice = Artist | AlbumArtist

artistRootPane : ArtistChoice -> MPane
artistRootPane artistChoice = ...
```

Not sure, it might even be nice to bring this to the protocol level (i.e., have an edn keyword `:artist` / `:albumartist` in the config)

Reply:
Much better, indeed.